packages/core/webpack/config/loaders.js

```diff
@@ -72,7 +72,7 @@ module.exports = props => {
 
   if (process.env.NODE_ENV === "development") {
     babelPlugins.push(
-      require.resolve("react-hot-loader/babel"),
+      // require.resolve("react-hot-loader/babel"),
```

Comment:
does the new worker-loader break hot reloading? why is this commented?

Reply:
Yeah, it breaks the hot-reloading. :(
 I'm on it.